lib/membrane_rtc_engine/endpoints/hls_endpoint.ex

```diff
@@ -294,7 +343,12 @@ if Enum.all?(
           {merge_parent_specs(spec, hls_sink_spec), state}
         end
 
-      {{:ok, spec: spec}, state}
+      update_action =
+        if is_nil(state.mixer_config) or track.type == :audio,
```

Comment:
The same condition is used in line 287, maybe we could move it to function with proper naming, like `compositor_track?`?